back/scripts/communities/loaders/odf.py

```diff
@@ -12,24 +13,32 @@ class OdfLoader:
     Data from OpenDataFrance, data.gouv.fr, 2022.
     This dataset lists the platforms and organizations that contribute to the development of open data in the territories, identified during the 2022 edition (as of December 31).
 
-    TODO : Refactor using loaders_factory
+    TODO : Refactor using loaders_factory when loading from disk
     """
 
     def __init__(self, config):
-        data_folder = Path(get_project_base_path()) / config["processed_data"]["path"]
-        data_file = data_folder / config["processed_data"]["filename"]
-        if data_file.exists():
-            self.data = pd.read_csv(data_file, sep=";")
-        else:
-            odf_data_loader = BaseLoader.loader_factory(config["url"], dtype=config["dtype"])
-            odf_data = odf_data_loader.load()
-            self.data = odf_data
-            self.save(
-                Path(config["processed_data"]["path"]), config["processed_data"]["filename"]
-            )
+        self._config = config
+        self._logger = logging.getLogger(__name__)
 
     def get(self):
-        return self.data
+        processed_data_config = self._config["processed_data"]
+        data_folder = Path(get_project_base_path()) / processed_data_config["path"]
+        data_file = data_folder / processed_data_config["filename"]
+
+        if data_file.exists():
+            self._logger.info("Found ODF file on disk, loading it.")
+            return pd.read_csv(data_file, sep=";")
 
-    def save(self, path, filename):
-        save_csv(self.data, path, filename, sep=";", index=True)
+        self._logger.info("Loading ODF data.")
+        odf_data_loader = BaseLoader.loader_factory(
+            self._config["url"], dtype=self._config["dtype"]
+        )
+        data = odf_data_loader.load()
+        save_csv(
+            data,
+            Path(processed_data_config["path"]),
```

Comment:
nit: let's use an absolute path
```suggestion
            data_folder,
```